src/controllers/UsersController.php

```diff
@@ -1245,10 +1245,9 @@ public function actionSaveUser()
         }
 
         // Manually validate the user so we can pass $clearErrors=false
-        if (
-            !$user->validate(null, false) ||
-            !Craft::$app->getElements()->saveElement($user, false)
-        ) {
```

Comment:
The way it was, it would call saveElement (w/ validation) when `validate` failed.